test/intl402/Locale/constructor-options-firstDayOfWeek-valid.js

```diff
@@ -50,6 +47,24 @@ const validFirstDayOfWeekOptions = [
   [6, "en-u-fw-sat"],
   [7, "en-u-fw-sun"],
   [0, "en-u-fw-sun"],
+  [true, "en-u-fw-true"],
```

Comment:
@FrankYFTang is this one correct? IIRC `true` is dropped from canonical locales. And that's how v8, SpiderMonkey, and LibJS all behave:
```js
> new Intl.Locale('en-u-fw-true').toString()
"en-u-fw" 

> new Intl.Locale('en-u-fw-false').toString()
"en-u-fw-false"
```

Reply:
I didn't see that in [CanonicalizeUValue](https://tc39.es/ecma402/#sec-canonicalizeuvalue), where should I be looking for that behvaiour?

Reply:
It would fall under step 2, which defers to TR-35.

It's in TR-35: https://unicode.org/reports/tr35/#Canonical_Unicode_Locale_Identifiers
> A unicode_locale_id has canonical syntax when:
> * Any type or tfield value "true" is removed.

Reply:
ah... that is true